mgmt/rest/v2/api.go

```diff
@@ -0,0 +1,96 @@
+package v2
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"sync"
+
+	"net/http"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/intelsdi-x/snap/core/api"
+	"github.com/urfave/negroni"
+)
+
+const (
+	version = "v2"
+	prefix  = "/" + version
+)
+
+var (
+	restLogger     = log.WithField("_module", "_mgmt-rest-v2")
+	protocolPrefix = "http"
+)
+
+type V2 struct {
```

Comment:
Should this (and V1) be unexported to avoid instantiation outside of calls to New? Maybe renamed also to avoid confusion around type V2 inside package v2?